presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/CassandraQueryRunner.java

```diff
@@ -30,32 +30,25 @@
 {
     private CassandraQueryRunner() {}
 
-    private static boolean tpchLoaded;
-
-    public static synchronized DistributedQueryRunner createCassandraQueryRunner()
+    public static DistributedQueryRunner createCassandraQueryRunner(CassandraServer server)
             throws Exception
     {
-        CassandraServer.start();
-
         DistributedQueryRunner queryRunner = new DistributedQueryRunner(createCassandraSession("tpch"), 4);
 
         queryRunner.installPlugin(new TpchPlugin());
         queryRunner.createCatalog("tpch", "tpch");
 
         queryRunner.installPlugin(new CassandraPlugin());
         queryRunner.createCatalog("cassandra", "cassandra", ImmutableMap.of(
-                "cassandra.contact-points", CassandraServer.getHost(),
-                "cassandra.native-protocol-port", Integer.toString(CassandraServer.getPort()),
+                "cassandra.contact-points", server.getHost(),
+                "cassandra.native-protocol-port", Integer.toString(server.getPort()),
                 "cassandra.allow-drop-table", "true"));
 
-        if (!tpchLoaded) {
-            createKeyspace(CassandraServer.getSession(), "tpch");
-            List<TpchTable<?>> tables = TpchTable.getTables();
-            copyTpchTables(queryRunner, "tpch", TINY_SCHEMA_NAME, createCassandraSession("tpch"), tables);
-            for (TpchTable<?> table : tables) {
-                CassandraServer.refreshSizeEstimates("tpch", table.getTableName());
-            }
-            tpchLoaded = true;
+        createKeyspace(server.getSession(), "tpch");
+        List<TpchTable<?>> tables = TpchTable.getTables();
```

Comment:
We can probably save some time by limiting this to only the tables we need (some test classes need 1-2 tables). This is how we do for eg Postgres.

(no change requested; might be a follow-up)